packages/hooking/manager.ts

```diff
@@ -248,28 +248,76 @@ export function registerAfterHook(
 	hookManager.registerHook(HookType.After, target, pkg, async, hookFn);
 }
 
+async function getFunction(
+	module: object,
+	propertyAccessors: string[],
+): Promise<unknown> {
+	let current = module;
+	for (const propertyAccessor of propertyAccessors) {
+		// @ts-ignore
+		current = current[propertyAccessor];
```

Comment:
I was wondering why it didn't throw any error, since I messed up the path to Socket.connect... before I got it working!
`finalizeHooks` wraps the call to `hookBuiltinFunction` in a try/catch already, and I wonder why. I will investigate!